legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/resources/BackupWorkspaceEntitiesResource.java

```diff
@@ -70,6 +70,31 @@ public List<Entity> getAllEntities(@PathParam("projectId") String projectId,
         );
     }
 
+    @GET
+    @Path("invalidEntities")
+    @ApiOperation("Get invalid entities of the backup user workspace")
+    public List<Entity> getInvalidEntities(@PathParam("projectId") String projectId,
```

Comment:
As with EntityAccessContext, this cannot return Entity. And I don't think the classifierPaths, stereotypes, or taggedValues parameters make sense since they rely on being able to deserialize the entity.